src/wp-includes/rest-api/endpoints/class-wp-rest-template-autosaves-controller.php

```diff
@@ -196,6 +196,34 @@ public function prepare_item_for_response( $item, $request ) {
 		return $response;
 	}
 
+	/**
+	 * Get the autosave, if the ID is valid.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_Post|WP_Error Revision post object if ID is valid, WP_Error otherwise.
+	 */
+	public function get_item( $request ) {
+		$parent = $this->get_parent( $request->get_param('parent') );
+		if ( is_wp_error( $parent ) ) {
+			return $parent;
+		}
+
+		$autosave = wp_get_post_autosave( $parent->ID );
+
+		if ( ! $autosave ) {
+			return new WP_Error(
+				'rest_post_no_autosave',
+				__( 'There is no autosave revision for this post.' ),
```

Comment:
"There is no autosave revision for this template"

Reply:
Fixed in e81bdc015d83e8674d22871978fe59cf0d877013